src/common/util.ts

```diff
@@ -27,9 +28,12 @@ export function filtersToWhere<C extends Colmuns>(filters: Filter<C>[]) {
         .reduce(
             (cond, filter) =>
                 cond +
-                `${format.ident(filter.column)} = ${format.literal(
-                    filter.value
-                )} AND `,
+                `${format.ident(filter.column)} ${
+                    filter.comparison &&
+                    ["=", "<", ">"].includes(filter.comparison)
```

Comment:
If `filter.comparison` is of type `"=" | "<" | ">"`, won't `["=", "<", ">"].includes(filter.comparison)` always be true?

Reply:
Not if it's ran as javascript: the TS code will compile fine, but then if the API is called with any random value there will be no JS runtime error (at least I don't think so)

Reply:
You're right. If you'll need to reuse this logic in the future, you can refactor it to a [type guard](https://www.typescriptlang.org/docs/handbook/advanced-types.html) function and replace
```
filter.comparison &&
["=", "<", ">"].includes(filter.comparison)
``` 
with `isComparisonOperator(filter.comparison)`